keyboards/dz60/keymaps/bingocaller/keymap.c

```diff
@@ -0,0 +1,115 @@
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    WORD_BACK = SAFE_RANGE,
+    WORD_FORWARD,
+    DELETE_WORD_BACK,
+    DELETE_WORD_FORWARD,
+    FINE_VOLUP,
+    FINE_VOLDOWN
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+        switch (keycode) {
+            case WORD_BACK:
+                SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_LEFT) SS_UP(X_LALT));
```

Comment:
You don't need to use `process_record_user` at all for these. You can simplify them by just doing:

```c
#define WORD_BACK           A(KC_LEFT)
#define WORD_FORWARD        A(KC_RIGHT)
#define DELETE_WORD_BACK    A(KC_BSPACE)
#define DELETE_WORD_FORWARD A(KC_DELETE)
#define FINE_VOLUP          S(A(KC__VOLUP))
#define FINE_VOLDOWN        S(A(KC__VOLDOWN))
```

Reply:
Ah, that’s so much simpler! It's been a while since I wrote those definitions but I remember thinking “There’s got to be a more straight-forward way of doing this…” 😄
I should still keep the `custom_keycodes` enum, right?

Reply:
The enum should be removed too otherwise it will conflict with the `#define`s.

Reply:
Alright, I’ll try it out; thanks! 🙂